src/storybook/story-test-runner/index.ts

```diff
@@ -0,0 +1,58 @@
+import { extendStoriesFromStoryFile } from "./extend-stories";
+import { nestedDescribe, extendedIt } from "./test-decorators";
+import { openStory } from "./open-story";
+import type { StoryLoadResult } from "./open-story/hermione-open-story";
+import type { HermioneOpts } from "../story-to-test";
+import type { TestFunctionExtendedCtx } from "../../types";
+import type { StorybookStoryExtended, StorybookStory } from "./types";
+
+export function getAbsoluteFilePath(): string {
```

Comment:
We need to pass story-test-runner absolute path to generated tests, so they can properly require it